pyogrio/tests/test_geopandas_io.py

```diff
@@ -167,17 +170,69 @@ def test_read_layer_invalid(naturalearth_lowres_all_ext, use_arrow):
 
 
 @pytest.mark.filterwarnings("ignore: Measured")
-def test_read_datetime(test_fgdb_vsi, use_arrow):
-    df = read_dataframe(
-        test_fgdb_vsi, layer="test_lines", use_arrow=use_arrow, max_features=1
-    )
-    if Version(pd.__version__) >= Version("2.0.0"):
+def test_read_datetime(test_fgdb_vsi):
+    df = read_dataframe(test_fgdb_vsi, layer="test_lines", max_features=1)
```

Comment:
```suggestion
def test_read_datetime(test_fgdb_vsi, use_arrow):
    df = read_dataframe(
        test_fgdb_vsi, layer="test_lines", use_arrow=use_arrow, max_features=1
    )
```

(essentially reverting the change on the lines above, this might be from a merge conflict)